junebug/command_line.py

```diff
@@ -82,6 +82,10 @@ def create_parser():
         help='Add a plugins to the list of plugins, as a json blob of the '
         'plugin config. Must contain a `type` key, with the full python class '
         'path of the plugin')
+    parser.add_argument(
+        '--metric-window', '-mw', type=float,
+        dest='metric_window', help='The size of each bucket '
+        '(in seconds) to use for metrics. Defaults to 1 second.')
```

Comment:
I'm not sure, what do we use in other places? The window really depends on how fast you're sending messages, the faster the shorter you want the window to be.